src/pages/Auth/SignUp.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Auth() {
+  return (
+    <SignUpWrapper>
```

Comment:
```suggestion
    <Container>
```
바깥쪽 요소는 Container로 통일해주세요